server/model/assessment.js

```diff
@@ -12,7 +12,7 @@ function assessment() {
     this.assessmentDescription = "";
     this.preThreshold = 90;             // default value that if acheived in pre assessment allows a student to "test-out" of topic
     this.postThreshold = 70;            // default value that if acheived in post assessment means a user can "move-on" from a topic
-    this.isRequired = "";
+    this.isRequired = false;
```

Comment:
Based on @briangormanly 's feedback in class today, we should leave this as an empty string and adjust at a later time if needed.